base_marker_id: Channel703
head_marker_id: Channel704
freq: 50
kp_x: 0.4
kp_y: 0.4
kp_z: 0.4
kp_r: 0.0
kp_p: 0.0
kp_w: 0.0
ki_x: 0.01
ki_y: 0.01
ki_z: 0.01
ki_r: 0.0
ki_p: 0.0
ki_w: 0.0
kd_x: 0.2
kd_y: 0.2
kd_z: 0.2
kd_r: 0.0
kd_p: 0.0
kd_w: 0.0
